draft-ietf-tls-esni.md

```diff
@@ -798,16 +775,18 @@ operation. Reusing the HPKE context avoids an attack described in
 The client then modifies the "encrypted_client_hello" extension in
 ClientHelloOuter as follows:
 
+- `type` is unchanged.
 - `config_id` is unchanged and contains the `config_id` corresponding to
   the client's chosen ECHConfig.
 - `cipher_suite` is unchanged and contains the client's chosen HPKE cipher
   suite.
 - `enc` is replaced with the empty string.
 - `payload` is replaced with the value computed above.
 
-If the client offered ECH in the first ClientHello, then it MUST offer ECH in
-the second. Likewise, if the client did not offer ECH in the first ClientHello,
-then it MUST NOT not offer ECH in the second.
+#### Rejected ECH
+
+The client constructs the updated ClientHelloOuter per the rules of {{RFC8446}}.
+In particular, the payload of the "encrypted_client_hello" does not change.
```

Comment:
Do we want to note this as a possible distinguisher here (or in the security considerations)?